api-schema/src/main/java/org/stellar/anchor/api/rpc/action/NotifyOffchainFundsReceivedRequest.java

```diff
@@ -0,0 +1,20 @@
+package org.stellar.anchor.api.rpc.action;
+
+import java.time.Instant;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class NotifyOffchainFundsReceivedRequest extends RpcActionParamsRequest {
+
+  private Instant fundsReceivedAt;
+  private String externalTransactionId;
+  private String amountIn;
```

Comment:
It's a bit unclear just from the naming when amount fields are just values and when it's object of value + asset, maybe we should update the naming?